### PR TITLE
Added wls logging exporter test

### PR DESCRIPTION
**Pull request by hzhao-github**

### CODE REVIEW

**Comment by hzhao-github** [comment]
Jenkins job passed

http://wls-jenkins.us.oracle.com/job/weblogic-kubernetes-operator-javatest/2042/consoleFull

File: integration-tests/src/test/java/oracle/kubernetes/operator/ItElasticLogging.java [hunk 1]
```diff
@@ -318,4 +389,96 @@ private String execElasticStackQuery(String queryCriteria) throws Exception {
 
     return result.stdout();
   }
+  
+  private void downloadWLSLoggingExporterJars() throws Exception {
+    // Location of Weblogic logging exporter jar files
+    final String loggingJarRepos = 
+      "https://github.com/oracle/weblogic-logging-exporter/releases/download/v0.1";
```
Comment (markxnelson) [inline integration-tests/src/test/java/oracle/kubernetes/operator/ItElasticLogging.java]: v0.1.1 is available now

File: integration-tests/src/test/resources/loggingexpdomainonpvwlst.yaml [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# Copyright 2018, 2019, Oracle Corporation and/or its affiliates.  All rights reserved.
```
Comment (markxnelson) [inline integration-tests/src/test/resources/loggingexpdomainonpvwlst.yaml]: is this a new file? remove "2018" from copyright notice

File: integration-tests/src/test/resources/loggingexporter/create-domain-logging-exporter.py [hunk 3]
```diff
@@ -0,0 +1,154 @@
+# Copyright 2018, 2019, Oracle Corporation and/or its affiliates.  All rights reserved.
```
Comment (markxnelson) [inline integration-tests/src/test/resources/loggingexporter/create-domain-logging-exporter.py]: new file? remove 2018

**Comment by hzhao-github** [comment]
4 test case3s in ItElasticLogging passed in Jenkins job:

http://wls-jenkins.us.oracle.com/job/weblogic-kubernetes-operator-javatest/2062/console

[INFO] Tests run: 4, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 521.867 s - in oracle.kubernetes.operator.ItElasticLogging

Failure was from:

[ERROR] Failures: 
[ERROR]   ItMonitoringExporter.test19_EndToEndViaChart:825->installPrometheusGrafanaViaChart:1144
[ERROR] Errors: 
[ERROR]   ItOperator.testTwoDomainsManagedByTwoOperators:230 » NullPointer

Thanks has nothing to do with ItElasticLogging

File: integration-tests/src/test/java/oracle/kubernetes/operator/ItElasticLogging.java (hunk 1 above)
Comment (hzhao-github) [reply integration-tests/src/test/java/oracle/kubernetes/operator/ItElasticLogging.java]: changed to:

  private final String loggingJarRepos = 
    "https://github.com/oracle/weblogic-logging-exporter/releases/download/v0.1.1";
  private final String wlsLoggingExpJar = "weblogic-logging-exporter-0.1.1.jar";

and the test downloaded it successfully

File: integration-tests/src/test/resources/loggingexpdomainonpvwlst.yaml (hunk 2 above)
Comment (hzhao-github) [reply integration-tests/src/test/resources/loggingexpdomainonpvwlst.yaml]: "2018" is removed

File: integration-tests/src/test/resources/loggingexporter/create-domain-logging-exporter.py (hunk 3 above)
Comment (hzhao-github) [reply integration-tests/src/test/resources/loggingexporter/create-domain-logging-exporter.py]: "2018" is removed

**Comment by hzhao-github** [comment]
Jenkins job kicked off

http://wls-jenkins.us.oracle.com/job/weblogic-kubernetes-operator-javatest/2072/

**Comment by hzhao-github** [comment]
Jenkins job passed again with latest changes

http://wls-jenkins.us.oracle.com/job/weblogic-kubernetes-operator-javatest/2090/consoleFull

[INFO] Tests run: 4, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 488.559 s - in oracle.kubernetes.operator.ItElasticLogging

**Comment by hzhao-github** [comment]
Jenkins job passed after synced-up with develop branch 

http://wls-jenkins.us.oracle.com/job/weblogic-kubernetes-operator-javatest/2135/console

**Comment by markxnelson** [comment]
Lgtm

**Pull request merged**